chain/byteexec/byteexec.go

```diff
@@ -33,21 +35,33 @@ func writeFile(executable []byte) (string, string, error) {
 }
 
 // RunOnLocal spawns a new subprocess and runs the given executable. NOT SAFE!
-func RunOnLocal(executable []byte, timeOut time.Duration, args ...string) ([]byte, error) {
+func RunOnLocal(executable []byte, timeOut time.Duration, arg string) ([]byte, error) {
+	args, err := shellwords.Parse(arg)
+	if err != nil {
+		return []byte{}, err
```

Comment:
@taobun. I think return `nil` makes more sense. Thoughts?

Reply:
In my opinion, I prefer `nil`. But other codes in our repo use `[]byte{}`. How we will fix it?

Reply:
Ok. let's use `nil` here then. We will deal with other places later.